src/main/java/org/springframework/data/couchbase/repository/support/CouchbaseRepositoryBase.java

```diff
@@ -0,0 +1,130 @@
+package org.springframework.data.couchbase.repository.support;
```

Comment:
Factor out common components of CouchbaseRepository and ReactiveCouchbaseRepository